pipeline/user_scripts/latest_reconstructions.py

```diff
@@ -24,28 +24,59 @@ def scan_all(*args, **kwargs):
 
 
 def main():
-    parser = argparse.ArgumentParser(description="utility for downloading the latest final (transformed) swc for each reconstruction")
-    parser.add_argument("runs_table_name", type=str, help="name of the dynamodb table listing all successful pipeline runs")
-    parser.add_argument("results_path", type=str, help="results will be written into this directory")
+    parser = argparse.ArgumentParser(
+        description="downloads the latest final (transformed) swc "
```

Comment:
Should probably clarify, that the script downloads the latest transformed swc as well as the raw swc and a marker

Reply:
And that the names includes specimen id not the reconstruction id.